### ISSUE TITLE
One of the 5 updates commits from yesterday broke searx (it just fails to search)

### ISSUE THREAD

**Issue by r3pek** [searx/searx#3063]
**Version of Searx, commit number if you are using on master branch and stipulate if you forked Searx**

It was this one (bd9c03b4) at the time I opened this:
**How did you install Searx?**

searx-docker
**What happened?**

Can't really pinpoint which commit borked it without testing all the 5 tags that exist but search itself stopped working and it just display an error "popup" saying "Failed to search".

**How To Reproduce**

Just the bd9c03b4 version and try to search something

**Expected behavior**

Search should work ;)

**Screenshots & Logs**

**Additional context**

I'll test it tomorrow which commit might have gone wrong if you don't find it meanwhile. 
EDIT: container logs don't show any error but i didn't enable any debug options. reverted to a880920d and everything is back to normal

**Comment by fabsau** [comment]
same for me :)
Edit: Tested all 5 images, none of them work. Hence it might be the first commit 8c3454fd causing this bug.

**Comment by poltrian** [comment]
Same for me on Docker

**Comment by Happyfeet01** [comment]
Same here

**Comment by zaggynl** [comment]
And my axe!

Workaround as suggested by r3pek:
(clean up existing folder)
git clone https://github.com/searx/searx.git
cd searx/
git reset --hard a880920
make docker
(create/use container as usual)

**Comment by 0xhtml** [comment]
Author of commit 8c3454fd1be4b578db5693ff1e939af238da6ac8 here.

Fix is out in pull request #3064.

As a work around one should be able to add the line
```yaml
prefer_configured_language: False
```
into the `search` section of the `settings.yml`.
e.g.
```yaml
search:
    ...
    prefer_configured_language: False
```

Sorry for breaking y'all's instance.
